lib/location/location_service_utils.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef LOCATION_SERVICE_UTILS_H
+#define LOCATION_SERVICE_UTILS_H
+
+/**
+ * @brief Provision TLS certificate that the selected location service requires
+ *	  for HTTPS connections.
+ *	  Certificate provisioning must be done before location requests can
+ *	  successfully be executed, either using this API or some other method.
+ *
+ * @note Certificate provisioning must happen when LTE is not active in the modem.
+ *	 This is typically achieved by calling this API prior to setting up the
+ *	 LTE link in an application, but it can also be done by setting the
+ *	 modem in offline mode before provisioning.
+ *
+ * @return 0 on success, or negative error code on failure.
+ */
+int location_service_utils_provision_certificates(void);
```

Comment:
a more accurate name would be `location_service_utils_provision_ca_certificate`

Reply:
This whole file has been removed now.

Reply:
And the file is back so renaming